backend/src/backend/app.py

```diff
@@ -115,6 +128,34 @@ async def get_tasks(user_id: Annotated[str, Depends(get_user_id)]):
     return [{"id": 1, "title": "Buy Milk"}, {"id": 2, "title": "Buy Bread"}]
 
 
+# Open AI chat
+openai.api_key = OPENAI_API_KEY 
+
+@app.post("/api/chat")
+async def chat(request: Request):
+    data = await request.json()
```

Comment:
In FastAPI, request body is defined to validate and enforce its structure. 
https://fastapi.tiangolo.com/tutorial/body/

How about we create a Pydantic model such as

```
class Chat(BaseModel):
  message: str
```

not sure about the requirements of this API endpoint tho

Reply:
by doing this, we don't need to write the validation at line 139